xmantissa/websession.py

```diff
@@ -277,16 +356,23 @@ def cbLoginSuccess(input):
                     self.savorSessionCookie(request)
             return input
 
-        return guard.SessionWrapper.login(self, request, session, creds, segments
-                                          ).addCallback(cbLoginSuccess)
+        return (
+            guard.SessionWrapper.login(
+                self, request, session, creds, segments)
+            .addCallback(cbLoginSuccess))
 
 
     def explicitLogout(self, session):
```

Comment:
There don't seem to be any tests (not in `test_websession` anyway) for the explicit log out case. Did I miss these somewhere?